src/components/accordion-disclosure/types.ts

```diff
@@ -13,7 +13,7 @@ export interface AccordionDisclosureProps {
 	 * emphasis. Always shows regardless of the `AccordionDisclosure`'s
 	 * open/closed state.
 	 */
-	description?: string
+	description?: string | ReactNode
```

Comment:
`string` is included in `ReactNode`:
```suggestion
	description?: ReactNode
```